src/features/query-bar/validator.js

```diff
@@ -1,15 +1,19 @@
 // @flow
+import { isInteger, isNaN } from 'lodash';
+
 import { FLOAT } from './constants';
 
-const floatRegex = /[^$,.\d]/;
+const isFloat = (value: string) => {
+    return !/^\s*$/.test(value) && !isNaN(Number(value));
+};
 
-const floatValidator = (value: string) => {
-    return value.match(floatRegex);
+const isInt = (value: string) => {
```

Comment:
Why is this function necessary? _.isInteger

Reply:
Will remove this method

Reply:
I agree. This doesn't seem to justify a separate utility.